### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by SantiSite**
### Description

Added the home page with 3 sections, the hero section, the testimonials section and the prices section. Also added a hamburger menu on the home page.

### CODE REVIEW

File: src/components/pages/Home/HowItWorks.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  });
+ }, []);
+
+ return (
+  <section
+   id="how-it-works"
+   aria-label="Features for running your books"
+   className="relative overflow-hidden pt-20 pb-10"
+  >
+   <img
```
Comment (odarbelaeze) [inline src/components/pages/Home/HowItWorks.tsx:28]: Let's remove this solid color image and associated code (the relative and absolute classes). Instead use `bg-leaf` on the `how-it-works` section.

File: src/components/pages/Home/HowItWorks.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    return curr;
+   }
+   return label;
+  });
+ }, []);
+
+ return (
+  <section
+   id="how-it-works"
+   aria-label="Features for running your books"
```
Comment (odarbelaeze) [inline src/components/pages/Home/HowItWorks.tsx:25]: Let's change this aria labe to `How Tree of Science works`.

Comment (SantiSite) [reply src/components/pages/Home/HowItWorks.tsx]: I have already done the change :dog:

File: src/components/pages/Home/HowItWorks.tsx (hunk 1 above)
Comment (SantiSite) [reply src/components/pages/Home/HowItWorks.tsx]: Perfect! :heart:

**Review by odarbelaeze — APPROVED** [review]
:ship::it:

**Pull request merged**